<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>466</height>
   </rect>
  </property>
  <property name="cursor" >
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle" >
   <string>SimAff</string>
  </property>
  <property name="dockNestingEnabled" >
   <bool>false</bool>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="tabPosition" >
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Tab 1</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_system" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionClose" />
    <addaction name="separator" />
    <addaction name="actionExit" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menu_View" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVisual" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>Vi&amp;sual</string>
    </property>
    <addaction name="actionNewVisual" />
    <addaction name="actionClearAllVisuals" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_View" />
   <addaction name="menuVisual" />
   <addaction name="menuHelp" />
  </widget>
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSystem_Description" >
   <property name="text" >
    <string>System Description</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionNewVisual" >
   <property name="text" >
    <string>&amp;New Visual</string>
   </property>
  </action>
  <action name="actionClear_All_Visuals" >
   <property name="text" >
    <string>Clear All Visuals</string>
   </property>
  </action>
  <action name="actionClearAllVisuals" >
   <property name="text" >
    <string>&amp;Clear All Visuals</string>
   </property>
  </action>
  <action name="actionNew_system" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;New system</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
